### PR TITLE
UI Changes, Landing Page, About, Library, New

### PR DESCRIPTION
**Pull request by Tandid**

### CODE REVIEW

File: app/layouts/default.vue [hunk 1]
```diff
@@ -1,12 +1,14 @@
 <template>
-  <div>
+  <div class="flex flex-col min-h-screen">
     <Navbar />
-    <main class="p-5">
+    <main class="flex-grow p-5">
```
Comment (davideastmond) [inline app/layouts/default.vue:4]: I was thinking the main can take up the entire width of the viewport on small screens, and we can add some margin on larger screens?

File: app/components/UserLibraryList.vue [hunk 2]
```diff
@@ -1,81 +1,107 @@
...
+      <div class="flex searchTextField lightRoundedGreyBorder">
+        <div class="self-center mt-2 ml-2">
           <Icon name="mdi:magnify" color="black" size="1.5rem" />
         </div>
-        <input type="text" placeholder="Search" class="searchInput w-full p-2 border border-black" />
+        <input
+          type="text"
+          placeholder="Search"
+          class="w-full p-2 border border-black searchInput"
+        />
```
Comment (davideastmond) [inline app/components/UserLibraryList.vue:30]: For the search and the dropdown, on smaller sizes, I'm thinking they should appear full-width in a column?

File: app/components/UserLibraryList.vue [hunk 3]
```diff
@@ -1,81 +1,107 @@
...
+        :key="doc.document_id"
+        :class="[
+          'p-4 hover:custom-green duration-200',
+          index % 2 === 0 ? 'bg-gray-100' : 'bg-white',
+        ]"
+      >
+        <div class="flex items-center justify-between">
           <div>
-            <p class="truncatable-text">
+            <p>
```
Comment (davideastmond) [inline app/components/UserLibraryList.vue:64]: In the Figma there is a document icon on the left and next to the title for each doc? 
There's also the `visibility` and the number of `stickies` - is that something you can address in this PR?

Comment (Tandid) [reply app/components/UserLibraryList.vue:69]: Updated this just now. Not sure how to access the number of stickies that currently exist in the doc though. Looks like I have to count the number of blocks with a cdx marker class?

File: app/components/UserLibraryList.vue (hunk 2 above)
Comment (Tandid) [reply app/components/UserLibraryList.vue:34]: Sounds good to me

File: app/layouts/default.vue (hunk 1 above)
Comment (Tandid) [reply app/layouts/default.vue:4]: I agree with this as well

File: app/components/UserLibraryList.vue (hunk 3 above)
Comment (davideastmond) [reply app/components/UserLibraryList.vue]: You can get all of the stickies by document_id and count them

**Review by davideastmond — APPROVED** [review]

**Pull request merged**